javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,49 @@
+//A future age calculator
+const yearOfBirth = 1994;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("\"You will be "+ age +" years old in 2027\"" );
+
+//A dog age calculator
+
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2025;
+const dogYearInHumanYear = dogYearFuture - dogYearOfBirth;
+const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+const inHumanYear = '\"your dog will be '+dogYearInHumanYear+' human year old in 2025\" ';
+const inDogYear =  '\"your dog will be '+dogYear+' dog year old in 2025\" ';
```

Comment:
'\"your dog will be '+dogYearInHumanYear+' human year old in 2025\" ';
What happen if I change the _const dogYearFuture = 2025;_ with _const dogYearFuture = 2035;_
You need to pass variable name instead hard coded text. So when ever you change the value, it will change every where.

Reply:
I have updated this file according to your feedback.
Thanks for your valuable comment.